exoscale/api/compute.py

```diff
@@ -1165,14 +1215,12 @@ def instances(self):
             latency.
         """
 
-        try:
-            [res] = self.compute.cs.getInstancePool(
-                id=self.id, zoneid=self.zone.id, fetch_list=True
-            )
-            for i in res.get("virtualmachines", []):
-                yield Instance._from_cs(self.compute, i, zone=self.zone)
-        except CloudStackApiException as e:
-            raise APIException(e.error["errortext"], e.error)
+        res = self.compute._v2_request(
```

Comment:
I was surprised to see that `API.send` (called by `_v2_request`) is a `requests.Requests` and not a call to an OpenAPI client.
Any reason not to use an OpenAPI client ?
(sorry I am late to the party)

Reply:
Yes, because the code generated using the OAS code generators I've tried produce more than double the amount of code of the entire library current codebase.